src/main/scala/is/hail/expr/AST.scala

```diff
@@ -236,7 +236,7 @@ sealed abstract class AST(pos: Position, subexprs: Array[AST] = Array.empty) {
   def runAggregator(ec: EvalContext): CPS[Any] = {
     val typedNames = ec.st.toSeq
       .sortBy { case (_, (i, _)) => i }
-      .map { case (name, (_, typ)) => (name, typ) }
+      .map { case (name, (_, typ)) => (name, typ.scalaClassTag.asInstanceOf[ClassTag[AnyRef]]) }
```

Comment:
If we have an environment of the form `Map[Name, F[T]]` we cannot choose `T` to be `_` because, in Scala, the existential is bound at the edges of the type expression:
```
∃T. Map[Name, F[T]]
```
It is impossible to add key-value pairs to this map because the consumer doesn't know what `T` is.

What we want is:

```
Map[Name, ∃T. F[T]]
```

I think the only way to write this in Scala is:

```
trait Existential {
  type T
  def thing: T
}

def addElement(m: Map[String, Existential]): Map[String, Existential] =
  m + ("foo" -> new Existential() { val thing = 3 })
```

This seemed like a whole lot of nonsense for not much value.